packages/editor-ui/src/components/WorkflowSettings.vue

```diff
@@ -566,9 +565,9 @@ export default defineComponent({
 	},
 	methods: {
 		onCallerIdsInput(str: string) {
-			this.workflowSettings.callerIds = /^[0-9,\s]+$/.test(str)
+			this.workflowSettings.callerIds = /^[a-zA-Z0-9,\s]+$/.test(str)
```

Comment:
nitpick - maybe this could be a constant then? instead of this being a magic value and have to update it in two places
```suggestion
			this.workflowSettings.callerIds = WORKFLOW_NANO_ID_REGEX.test(str)
```

Reply:
They are slightly different, one contains the `g` modifier at the end, so I don't know if we can combine them.

Reply:
ah missed that.. you can if you use new RegExp I guess but not a big deal